.travis/linux_runner.sh

```diff
@@ -66,15 +66,22 @@ do_install() {
     ls -l ./
     if [ ! -f "build-cache/grpc_server_example" ]; then
         sudo apt-get install golang
-
         git clone https://github.com/iresty/grpc_server_example.git grpc_server_example
-
         cd grpc_server_example/
         go build -o grpc_server_example main.go
         mv grpc_server_example ../build-cache/
+        mv proto/ ../build-cache/
```

Comment:
we need to do a clean after this PR merged, because it shared a same flag with `grpc_server_example`.